use-cases/using-SRIOV-and-kata.md

```diff
@@ -316,3 +317,31 @@ The following example launches a Kata Containers container using SR-IOV:
    ```
    sriov-2:~$ sudo docker run --runtime=kata-runtime --net=vfnet --cap-add SYS_ADMIN --ip=192.168.0.11 -it mcastelino/iperf iperf3 -c 192.168.0.10 bash -c "mount -t ramfs -o size=20M ramfs /tmp; iperf3 -c 192.168.0.10"
    ```
+## Example 2: Launch a Kata Containers container using SR-IOV without SR-IOV Plugin
+
+The second example launches a Kata Containers container using SR-IOV, however doesn't needs an additional plugin. A self build guest kernel, which includes the appropriate NIC driver kernel module, might be necessary. 
+
+1. Get PCI address of VF:
+   ```
+   PCI=<PCI address of VF>
+   ```
+2. Get IOMMU group of VF:
+   ```
+   GROUP=$(readlink -e /sys/bus/pci/devices/$PCI/iommu_group | sed "s/.*\///")
+   ```
+   
+3. Unbind VF from current network driver (driver name might be different):
+   ```
+   echo "$PCI" > /sys/bus/pci/drivers/i40e/unbind
```

Comment:
This assumes the user is running as `root`. All our docs require that the user is not initially running as `root` (again see https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md), so this would need to be something like:

```sh
$ echo "$PCI" | sudo tee /sys/bus/pci/drivers/i40e/unbind
```